AutomatedTesting/Gem/PythonTests/prefab/Prefab_Test_Utils.py

```diff
@@ -0,0 +1,103 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+import os
+
+from azlmbr.entity import EntityId
+from azlmbr.math import Vector3
+from editor_python_test_tools.utils import Report
+from editor_python_test_tools.utils import TestHelper as helper
+
+import azlmbr.bus as bus
+import azlmbr.components as components
+import azlmbr.editor as editor
+import azlmbr.entity as entity
+import azlmbr.legacy.general as general
+
+
+class UtilsResults:
+    unique_name_entity_found = (
+        "Entity with a unique name found",
+        "Entity with a unique name *not* found")
+
+    prefab_at_expected_position = (
+        "prefab is at expected position",
+        "prefab is *not* at expected position")
+
+    entity_children_count_matched = (
+        "Entity with a unique name found",
+        "Entity with a unique name *not* found")
+
+
+def get_prefab_file_name(prefab_name):
+    return prefab_name + ".prefab"
+
+
+def get_prefab_file_path(prefab_name):
+    return os.path.join(os.path.dirname(os.path.abspath(__file__)), get_prefab_file_name(prefab_name))
+
+
+def find_entities_by_name(entity_name):
+    searchFilter = entity.SearchFilter()
+    searchFilter.names = [entity_name]
+    return entity.SearchBus(bus.Broadcast, 'SearchEntities', searchFilter)
+
+
+def find_entity_by_unique_name(entity_name):
+    entities = find_entities_by_name(entity_name)
+    unique_name_entity_found = len(entities) is 1
```

Comment:
this doesn't do what you think it does. Use == 1 instead

Reply:
Will fix it.

Reply:
Fixed.